tck/src/main/java/org/eclipse/microprofile/opentracing/tck/application/TestServerWebServices.java

```diff
@@ -405,6 +487,25 @@ private void executeNested(String requestUrl) {
         return target.request().async().get();
     }
 
+
+    public URL baseUrl() {
+        String incomingUrl = uri.getAbsolutePath().toString();
```

Comment:
Use of both `incomingUrl` and `incomingURL` are confusing.  Suggest to rename `incomingUrl` to something like `incomingUrlValue`.